diff --git a/node_modules/react-native-siri-shortcut/ios/RNSSAddToSiriButtonViewManager.m b/node_modules/react-native-siri-shortcut/ios/RNSSAddToSiriButtonViewManager.m
index 4705319..df21980 100644
--- a/node_modules/react-native-siri-shortcut/ios/RNSSAddToSiriButtonViewManager.m
+++ b/node_modules/react-native-siri-shortcut/ios/RNSSAddToSiriButtonViewManager.m
@@ -42,7 +42,7 @@ + (BOOL)requiresMainQueueSetup
 
 - (NSDictionary *)constantsToExport
 {
-    INUIAddVoiceShortcutButton *button = [INUIAddVoiceShortcutButton new];
+    INUIAddVoiceShortcutButton *button = [[INUIAddVoiceShortcutButton alloc] initWithStyle:INUIAddVoiceShortcutButtonStyleAutomatic];
     button.translatesAutoresizingMaskIntoConstraints = NO;
     [button layoutIfNeeded];
 
